dbms/src/Storages/Transaction/FastAddPeer.h

```diff
@@ -0,0 +1,52 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+namespace DB
+{
+struct TempUniversalPageStorage
+{
+    UniversalPageStoragePtr temp_ps;
+    std::vector<String> paths;
+
+    ~TempUniversalPageStorage()
+    {
+        for (const auto & path : paths)
+        {
+            Poco::File(path).remove(true);
+        }
+    }
+};
+using TempUniversalPageStoragePtr = std::shared_ptr<TempUniversalPageStorage>;
+
+struct AsyncTasks;
+
+struct FastAddPeerContext
+{
+    explicit FastAddPeerContext();
+    pingcap::pd::ClientPtr pd_client;
+    std::shared_ptr<AsyncTasks> tasks_trace;
+
+    std::mutex mu;
+    std::unordered_map<UInt64, std::pair<String, TempUniversalPageStoragePtr>> temp_ps_cache;
```

Comment:
Please add documents to explain the content of `temp_ps_cache`.

Reply:
Added.